client/config/environment.js

```diff
@@ -28,9 +28,19 @@ module.exports = function(environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+
+    'labs-search': {
+      host: 'https://search-api.planninglabs.nyc',
+      route: 'search',
+      helpers: ['geosearch', 'bbl'],
+    },
```

Comment:
Because you already this up here how come you need it in the other enviornments?

Reply:
Oh whoops, forgot to remove this. will do